WordPress/src/main/java/org/wordpress/android/ui/plans/util/IabHelper.java

```diff
@@ -169,18 +167,8 @@ public IabHelper(Context ctx, String base64PublicKey) {
         logDebug("IAB helper created.");
     }
 
-    /**
-     * Enables or disable debug logging through LogCat.
-     */
-    public void enableDebugLogging(boolean enable, String tag) {
-        checkNotDisposed();
-        mDebugLog = enable;
-        mDebugTag = tag;
-    }
-
     public void enableDebugLogging(boolean enable) {
         checkNotDisposed();
```

Comment:
Since `IabHelper` has been modified to always have debug logging, this method really isn't necessary. But I think it would be better to keep `IabHelper` the way it was and only change the `Log.` calls to `AppLog.` calls so there are fewer code changes (since this class is provided by Google and will likely change).